ReactiveXComponent/ReactiveXComponent/__init__.py

```diff
@@ -0,0 +1,29 @@
+__version__ = '1.0.0'
+
+import logging
+try:
+    # not available in python 2.6
+    from logging import NullHandler
+except ImportError:
+
+    class NullHandler(logging.Handler):
+
+        def emit(self, record):
+            pass
+
+# Add NullHandler to prevent logging warnings
+logging.getLogger(__name__).addHandler(NullHandler())
+
+from ReactiveXComponent.xcomponentAPI import xcomponentAPI
```

Comment:
It could be a nice idea to configure a tool such as https://www.pylint.org to enforce the coding style.
We're doing something similar in typescript with Hazem